start/routes.ts

```diff
@@ -18,8 +18,22 @@
 |
 */
 
-import Route from '@ioc:Adonis/Core/Route'
+import Route from '@ioc:Adonis/Core/Route';
 
 Route.get('/', async () => {
-  return { hello: 'world' }
-})
+  return { hello: 'Diwala' };
+});
+
+Route.group(() => {
+
+    //User Authentication
+    Route.post('/register', 'UsersController.register');
```

Comment:
Im a bit weary/scared that it is string based, is there no other way of doing this?
Meanight the controller and the method in the controller?

Reply:
Ideally, routes in adonis accept a route pattern as the first argument and a route handler as a second argument, however, a route handler can be bloated with too much code so I break down the route handler into a controller with a specific method that takes care of that logic which enforces clean code principles, and that is how Adonisjs does it as the path to controllers and their methods are string-based.

Reply:
What is your thoughts on it? @arnoldkunihira

Reply:
My thoughts are, the routes file is kept clean and this has these benefits:
1. It is easy to make changes in the code.
2. I can reuse the existing model, controller anywhere within the code hence it utilizes the DRY principle.

Reply:
Sounds good!